data/common/ubuntu-dev-qt

```diff
@@ -0,0 +1,14 @@
+# SPDX-FileCopyrightText: Andrew Hayzen <ahayzen@gmail.com>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+ENV DEBIAN_FRONTEND=noninteractive
+
```

Comment:
add an `example-qt-5-15` which has the `runargs` and `PATH` set and a readme explaining the expected Qt installer location.